packages/device_info_plus/lib/device_info_plus_tizen.dart

```diff
@@ -109,12 +109,33 @@ class TizenDeviceInfo {
       tizenId: map['tizenId'],
     );
   }
+
+  /// Device information data.
+  Map<String, dynamic> get data => {
+        'modelName': modelName,
+        'cpuArch': cpuArch,
+        'nativeApiVersion': nativeApiVersion,
+        'platformVersion': platformVersion,
+        'webApiVersion': webApiVersion,
+        'profile': profile,
+        'buildDate': buildDate,
+        'buildId': buildId,
+        'buildString': buildString,
+        'buildTime': buildTime,
+        'buildType': buildType,
+        'buildVariant': buildVariant,
+        'buildRelease': buildRelease,
+        'deviceType': deviceType,
+        'manufacturer': manufacturer,
+        'platformName': platformName,
+        'platformProcessor': platformProcessor,
+        'tizenId': tizenId,
+      };
```

Comment:
Can you explain where this data getter is used? I couldn't find out why this is needed.

Reply:
Thank you for explain. I missed PR comment.
I think we need to add the toString() API too.

https://github.com/fluttercommunity/plus_plugins/blob/a246de0323c454b0a89dfa0fc1ba6e7aa58ef477/packages/device_info_plus/device_info_plus_platform_interface/lib/model/base_device_info.dart#L13-L16

Reply:
Okay, let me clean up a bit.

Reply:
Done. Thanks!